ambry-api/src/main/java/com.github.ambry/messageformat/BlobProperties.java

```diff
@@ -23,17 +23,23 @@
  */
 public class BlobProperties {
 
+  public static final short ACCOUNTID_CONTAINERID_DEFAULT_VALUE = -1;
+
   protected long blobSize;
   protected String serviceId;
   protected String ownerId;
   protected String contentType;
   protected boolean isPrivate;
   protected long timeToLiveInSeconds;
   protected long creationTimeInMs;
+  private short accountId;
+  private short containerId;
+  private short issuerAccountId;
 
   /**
    * @param blobSize The size of the blob in bytes
    * @param serviceId The service id that is creating this blob
+   * @TODO: Remove this constructor once BlobProperty V2 is enabled
```

Comment:
Just for ease. In the next PR, when we start creating BlobProperties in V2, I can track these todos and take actions.